src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -23,6 +25,19 @@ enum CodeSnippetView {
   JSON = "json",
 }
 
+const labelRenderer: LabelRenderer = (keyPath) => {
+  const currentKey = keyPath[0];
+  // If this is a top level key then it is a model UUID.
+  if (keyPath.length === 1) {
+    return (
+      <ModelDetailsLink uuid={currentKey.toString()}>
```

Comment:
> Was wondering if maybe we could add an optional prop to `ModelDetailsLink` that would add an `onClick={(e) => e.stopPropagation()}` to the `Link` component within it. This is for the use-case when we open up a model in a new tab with command + click, as the current view for the model in the tree would either collapse/expand, but I think it would be more intuitive if it would collapse/expand only when we press the small arrow button next to it instead. This is only for the model, as for the children it makes sense to collapse/expand when pressing on them as well.

Ah yes, well spotted! I've fixed that.

> Also, I was wondering if maybe there could be situations where the user might only know the `modelUUID` without knowing the actual model name. For those cases, would it make sense to maybe display the `modelUUID` when hovering on the model name or displaying both the model name and `modelUUID` in order to make it easier for the user to command+F by `modelUUID`?

I've opted to display the UUID  as a `title="..."` tooltip instead of a Vanilla one as it felt a bit too intrusive, but maybe we can revisit this if users need it to be more obvious. I'm not sure how crucial the UUID is it doesn't get displayed to the user anywhere else in the UI.